lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -1663,7 +1658,9 @@ void LLVMToSPIRV::transGlobalAnnotation(GlobalVariable *V) {
 
 bool LLVMToSPIRV::transGlobalVariables() {
   for (auto I = M->global_begin(), E = M->global_end(); I != E; ++I) {
-    if (!transValue(&(*I), nullptr))
+    if ((&(*I))->getName() == "llvm.global.annotations")
```

Comment:
Would this work as well?
```suggestion
    if (((*I).getName() == "llvm.global.annotations")
```

Reply:
As it turns out, yes